LoopUI/ViewModel/CGMStatusHUDViewModel.swift

```diff
@@ -32,15 +32,12 @@ public class CGMStatusHUDViewModel {
         return manualGlucoseTrendIconOverride
     }
 
-    private var glucoseValueCurrent: Bool {
-        guard let isStaleAt = isStaleAt else { return true }
-        return Date() < isStaleAt
-    }
+    var isGlucoseValueCurrent: Bool = true
```

Comment:
Would prefer to keep the terminology as "stale", rather than "current".  The "current" bg could be stale.